app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
I'm with David on this one. `role=...` filter seems like a straightforward solution. We could only allow `role=buyer` right now (which would fit with the "APIs don't get smaller" but will be nicer to extend to other roles without breaking backwards compatibility), but for example finding all admin or admin-ccs users for the environment is something I had to do in the past, so an endpoint would definitely be useful.